db/seeds.rb.example

```diff
@@ -0,0 +1,20 @@
+# Replace <your gmail name> with your gmail name below.
+
+Community.create!([
+  {name: "My Community", abbrv: "MC"}
+])
+
+Household.create!([
+  {unit_num: "0", community_id: 1, name: "Admins", deactivated_at: nil}
+])
+
+User.create!([
+  {email: "<your gmail name>@gmail.com", google_email: "<your gmail name>@gmail.com", first_name: "Alice", last_name: "Admin", mobile_phone: "17345551212", household_id: 1, admin: true}
```

Comment:
Same thing here about guessing. Also this line is too long. Once nice way to format long hashes like this is to put each key/value pair on its own line.